pkg/cli/alpha/internal/update/prepare_test.go

```diff
@@ -0,0 +1,180 @@
+		err = os.Chdir(workDir)
+		Expect(err).ToNot(HaveOccurred())
+
+		err = os.RemoveAll(tmpDir)
+		Expect(err).ToNot(HaveOccurred())
+		defer gock.Off()
+	})
+
+	Context("Prepare", func() {
+		DescribeTable("should succeed for valid options",
+			func(options *Update) {
+				const version = `version: "3"`
+				Expect(os.WriteFile(projectFile, []byte(version), 0o644)).To(Succeed())
+
+				result := options.Prepare()
+				Expect(result).ToNot(HaveOccurred())
+				Expect(options.Prepare()).To(Succeed())
+				Expect(options.FromVersion).To(Equal("v1.0.0"))
+				Expect(options.ToVersion).To(Equal("v1.1.0"))
+			},
+			Entry("options", &Update{FromVersion: "v1.0.0", ToVersion: "v1.1.0", FromBranch: "test"}),
+			Entry("options", &Update{FromVersion: "1.0.0", ToVersion: "1.1.0", FromBranch: "test"}),
+			Entry("options", &Update{FromVersion: "v1.0.0", ToVersion: "v1.1.0"}),
+			Entry("options", &Update{}),
+		)
+
+		DescribeTable("Should fail to prepare if project path is undetermined",
+			func(options *Update) {
+				err = options.Prepare()
+				Expect(err).To(HaveOccurred())
+				Expect(err.Error()).Should(ContainSubstring("failed to determine project path"))
+			},
+			Entry("options", &Update{FromVersion: "v1.0.0", ToVersion: "v1.1.0", FromBranch: "test"}),
+		)
+
+		DescribeTable("Should fail if PROJECT config could not be loaded",
+			func(options *Update) {
+				const version = ""
+				Expect(os.WriteFile(projectFile, []byte(version), 0o644)).To(Succeed())
+
+				err := options.Prepare()
+				Expect(err).To(HaveOccurred())
+				Expect(err.Error()).Should(ContainSubstring("failed to load PROJECT config"))
+			},
+			Entry("options", &Update{FromVersion: "v1.0.0", ToVersion: "v1.1.0", FromBranch: "test"}),
+		)
+
+		DescribeTable("Should fail if FromVersion cannot be determined",
+			func(options *Update) {
+				config.Register(config.Version{Number: 3}, func() config.Config {
+					return &v3.Cfg{Version: config.Version{Number: 3}}
+				})
+
+				const version = `version: "3"`
+				Expect(os.WriteFile(projectFile, []byte(version), 0o644)).To(Succeed())
+				Expect(options.FromVersion).To(BeEquivalentTo(""))
+			},
+			Entry("options", &Update{}),
+		)
+	})
+
+	Context("DefineFromVersion", func() {
+		DescribeTable("Should succeed when branch or CliVersion in Project config is present",
+			func(options *Update) {
+				const version = `version: "3"`
+				Expect(os.WriteFile(projectFile, []byte(version), 0o644)).To(Succeed())
+
+				config, err := common.LoadProjectConfig(tmpDir)
+				Expect(err).ToNot(HaveOccurred())
+				fromVersion, err := options.defineFromVersion(config)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(fromVersion).To(BeEquivalentTo("v1.0.0"))
+			},
+			Entry("options", &Update{FromVersion: ""}),
+			Entry("options", &Update{FromBranch: "test"}),
+			Entry("options", &Update{FromVersion: "1.0.0", FromBranch: "test"}),
+		)
+		DescribeTable("Should fail when branch and CliVersion in Project config are absent",
```

Comment:
We should fail when cliVersion is not found in the project file and the --from-version option is not specified.
Could you please fix this one?